vm/rust/src/lib.rs

```diff
@@ -55,7 +53,19 @@ extern "C" {
     fn JunoAppendActualFee(reader_handle: usize, ptr: *const c_uchar);
 }
 
-const N_STEPS_FEE_WEIGHT: f64 = 0.005;
+const GLOBAL_CONTRACT_CACHE_SIZE: usize= 100; // Todo ? default used to set this to 100.
+
+
+
+use lazy_static::lazy_static;
+extern crate serde_json;
+use std::sync::Mutex;
+const VERSIONED_CONSTANTS_JSON: &'static str = include_str!("../versioned_constants.json");
+lazy_static! {
+    static ref VERSIONED_CONSTANTS: Mutex<Result<VersionedConstants, serde_json::Error>> = Mutex::new(serde_json::from_str(VERSIONED_CONSTANTS_JSON));
+}
+
```

Comment:
version constants are supposed to be read from the respective JSON according to the SN version that we are currently executing on.

for example; [versioned_constants_13_0.json](https://github.com/starkware-libs/blockifier/blob/main/crates/blockifier/resources/versioned_constants_13_0.json) should be used if the SN version of the block that we are executing on is 0.13.0

Reply:
Note for myself, [versioned_constants.json](https://github.com/starkware-libs/blockifier/blob/main/crates/blockifier/resources/versioned_constants.json#L458) and [versioned_constants_13_0.json](https://github.com/starkware-libs/blockifier/blob/main/crates/blockifier/resources/versioned_constants_13_0.json#L383) have different formats :/